lib/chef/resource/registry_key.rb

```diff
@@ -152,47 +150,36 @@ class RegistryKey < Chef::Resource
       # may want to extend the state_attrs API with the ability to rename POST'd attrs.
       #
       # See lib/chef/resource_reporter.rb for more information.
+      @unscrubbed_values = []
```

Comment:
this is creating a class variable, this could be turned into a lazy initializer:

```
def unscrubbed_values
  @unscrubbed_values ||= []
end
```